### PR TITLE
Add support for proxying websockets too

### LINKED ISSUE #6: Support websockets
Right now we do not support websockets.

We should!

### PR DESCRIPTION
**Pull request by yuvipanda**
Unfortunately this also bumps up minimum required Python version to 3.5
in the same commit. I'm ok with that, but should've ideally been a separate PR.  
  
Fixes #6

### CODE REVIEW

File: nbserverproxy/handlers.py [hunk 1]
```diff
@@ -5,18 +5,88 @@
...
+        Called when the client closes our websocket connection.
+
+        We close our connection to the backend too.
+        """
+        self.ws.close()
 
     @web.authenticated
-    @gen.coroutine
-    def proxy(self, port, proxied_path):
+    async def proxy(self, port, proxied_path):
```
Comment (betatim) [inline nbserverproxy/handlers.py]: what does this change to do the minimum python version required?

Comment (minrk) [reply nbserverproxy/handlers.py]: makes it Python 3.5

Comment (yuvipanda) [reply nbserverproxy/handlers.py]: I put that info in the commit message, forgot to put it in PR. Have done so now!

**Pull request merged**